doc/examples/gfal_copy.c

```diff
@@ -47,21 +47,24 @@ static void my_monitor_callback(gfalt_transfer_status_t h, const char* src,
         size_t inst = gfalt_copy_get_instant_baudrate(h, NULL);
         size_t trans = gfalt_copy_get_bytes_transfered(h, NULL);
         time_t elapsed = gfalt_copy_get_elapsed_time(h, NULL);
-
-        printf("%d bytes/second average (%d instant). Transferred %d, elapsed %d seconds", avg, inst, trans, elapsed);
+        printf("%d bytes/second average (%d instant). Transferred %d, elapsed %d seconds \n", avg, inst, trans, elapsed);
     }
 }
 
 
 int main(int argc, char** argv)
 {
+    printf("Main");
     if (argc < 3) {
         fprintf(stderr, "Missing source and/or destination parameter");
         abort();
     }
     const char* source = argv[1];
     const char* destination = argv[2];
 
+    setenv("SRC_URL", source, 1);
+    setenv("DST_URL", destination, 1);
+
```

Comment:
Here I would add a comment stating that
```
The authentication with Reva is yet to be properly designed. Nevertheless, there is a need to associate tokens to URLs given that a token issued for a destination must have write rights, whereas right now GFAL issues a `stat()` == `HEAD` request also to the destination without requiring write rights. To be further developed.
```

Reply:
I would revert even this. It's too specific to a protocol of a particular plugin to be part of the main `gfal_copy` example.

Reply:
I see your point. But then we need to document these env vars are required by the plugin -> in Davix?

Reply:
Either Davix or the [gfal_http_plugin.cpp](https://github.com/cern-fts/gfal2/blob/master/src/plugins/http/gfal_http_plugin.cpp) file.